boards/same54-xpro/Makefile.features

```diff
@@ -13,5 +14,10 @@ FEATURES_PROVIDED += periph_uart
 FEATURES_PROVIDED += periph_adc
 FEATURES_PROVIDED += periph_usbdev
 
+# enables sam0_eth as default network device
+ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
```

Comment:
doesn't work without gnrc_netdev_default.
Did you mean netdev_default instead ?

Reply:
huh that is odd - `gnrc_netdev_default` will [pull in](https://github.com/RIOT-OS/RIOT/blob/master/sys/Makefile.dep#L135) `netdev_default`

Reply:
Yes so I must remove netdev_default not gnrc_netdev_default :)

Reply:
Huh? If the application selects `gnrc_netdev_default` the build system should automatically select `netdev_default`

Reply:
Yes but in your commit suggestion, you suggests that I must remove `gnrc_netdev_default` but I guess you were wrong, I should remove `netdev_default` since it will be pull in by `gnrc_netdev_default`

Reply:
Yes but ifconfig shows nothing if I remove gnrc_netdev_default, maybe there is a bigger issue hidden ?

Reply:
Ah!
I found the issue - this belongs to `Makefile.dep`, not `Makefile.features`